default/stringtables/en.txt

```diff
@@ -3281,6 +3281,26 @@ Duplicates Existing Design
 DESIGN_KNOWN_DETAIL
 This design is a duplicate of "%1%".
 
+DESIGN_OBSOLETE
+Obsolete Design
+
+DESIGN_WND_ADD
+Add New Design
+
+DESIGN_WND_REPLACE
+Replace Old Design
+
+DESIGN_WND_REPLACE_DETAIL
+Replace "%1%"
```

Comment:
I don't understand the question exactly.

The first three are button texts.
The fourth is the tooltip text that tells the user which design will be replaced.
I put it in the tooltip so that the user can have long ship names and the button will remain a predictable length.